Formula/cassandra.rb

```diff
@@ -1,7 +1,7 @@
 class Cassandra < Formula
   desc "Eventually consistent, distributed key-value store"
   homepage "https://cassandra.apache.org"
-  url "https://www.apache.org/dyn/closer.cgi?path=cassandra/3.11.4/apache-cassandra-3.11.4-bin.tar.gz"
```

Comment:
Does the original URL format work? If you want to go with `archive.apache.org`, you might want to put it into `mirror` field.

Reply:
Nope, it seems they just discontinued this format and suggest to use the new one